api/handlers/container/container.go

```diff
@@ -22,9 +22,9 @@ type Service interface {
 	GetPathToFilesInContainer(ctx context.Context, cid string, path string) (string, func(), error)
 	Remove(ctx context.Context, cid string, force, removeVolumes bool) error
 	Wait(ctx context.Context, cid string, condition string) (code int64, err error)
-	Start(ctx context.Context, cid string) error
+	Start(ctx context.Context, containers []string, options ncTypes.ContainerStartOptions) error
```

Comment:
Does docker api support starting multiple containers?

Reply:
Pasting from another thread

Nerdctl currentl accepts container id's in [[]string format](https://github.com/containerd/nerdctl/blob/89003a58835a7907687ffb46d3d133349cfc7ce9/pkg/cmd/container/start.go#L31C60-L31C64).
I think making our functions take cid in input was the right approach, we can send the cid in the format that nerdctl expects when we make that call. 

Starting multiple containers is not a feature that docker supports at the moment - https://docs.docker.com/reference/api/engine/version/v1.43/#tag/Container/operation/ContainerStart

I will update this.